packages/scripts/src/helpers/config.ts

```diff
@@ -160,6 +163,14 @@ export const JEST_MAX_WORKERS = process.env.JEST_MAX_WORKERS
 
 export const NPM_DEFAULT_REGISTRY = 'https://registry.npmjs.org/';
 
+export const CERT_PATH = process.env.CERT_PATH
+    || ((ENCRYPT_KAFKA || ENCRYPT_MINIO || ENCRYPT_OPENSEARCH)
```

Comment:
Can we use the new ENCRYPTION_ENABLED var here instead?
```suggestion
    || (ENCRYPTION_ENABLED
```